apps/site/components/Blog/BlogHeader/index.tsx

```diff
@@ -21,6 +21,7 @@ const BlogHeader: FC<BlogHeaderProps> = ({ category }) => {
         {t('layouts.blog.title')}
         <Link
           href={`/feed/${currentFile}`}
+          {...{ locale: 'en' }} // RSS feeds only exist in English
```

Comment:
```suggestion
          locale='en' // RSS feeds only exist in English
```

Reply:
@avivkeller 
I used object spread here to pass locale because our custom Link component’s prop types are based on HTMLAnchorElement and don’t include locale, thus will cause type error. This approach avoids a larger refactor while keeping the behavior consistent with how the locale prop would be passed in a properly typed Next.js Link. It also ensures we don’t introduce unnecessary complexity just to satisfy type checking for a value that is already fixed (en).

<img width="1099" height="405" alt="截圖 2025-08-15 晚上9 06 08" src="https://github.com/user-attachments/assets/e7be3b25-5ccc-44a4-9c9d-b2b54ea9566e" />

Reply:
I can also do that change in my day time tomorrow.
I will see if there is an updated one tomorrow, or else I can help the change.
Thanks for the feedback @avivkeller